website/docs/tools/advisor.md

```diff
@@ -14,6 +14,25 @@ The providers require specific configuration in the [ORT configuration file](htt
 When executing the advisor, the providers to enable are selected with the `--advisors` option (or its short alias `-a`); here a comma-separated list with provider IDs is expected.
 The following sections describe the providers supported by the advisor:
 
+# Black Duck
+
+This vulnerability provider obtains information about security vulnerabilities from the Black Duck instance specified in the configuration.
+The configuration is mandatory, because authentication is required.
+The implementation is in *experimental* state.
+Initial experiments indicate that it works with the ecosystems crate, gem, hackage, maven, npm, nuget, pod, pub, and pypi, see https://github.com/oss-review-toolkit/ort/issues/9638.
```

Comment:
In order to not have to maintain this list over here, how about saying

"[...] it works with the ecosystems mentioned [over here](https://github.com/oss-review-toolkit/ort/issues/9638)."

Please also note that this goes to the website and as such probably should be less technical (also `#xxxx`-type of issue links to no get rendered that way, as the Markdown is not hosted on GitHub, but by Docusaurus).